handlers/parse/guest_link.go

```diff
@@ -1,15 +1,20 @@
 package parse
 
 import (
+	"errors"
 	"fmt"
+	"time"
 
 	"github.com/mtlynch/picoshare/v2/picoshare"
 )
 
 // Arbitrary limit to prevent too-long labels in the UI.
 const MaxGuestLinkLabelLength = 200
+const validTimeUnits = "1ns, 1us (or 1µs), 1ms, 1s, 1m, 1h"
 
 var ErrGuestLinkLabelTooLong = fmt.Errorf("label too long - limit %d characters", MaxGuestLinkLabelLength)
+var ErrFileLifeTimeUnrecognizedFormat = fmt.Errorf("unrecognized format for file life time, must be in %s format", validTimeUnits)
```

Comment:
Can we update the naming in this file so that it's `Lifetime` with lowercase `t` and `lifetime` (one word) in error messages?

Reply:
I hope I got them all now... Sorry for that :D